src/components/SetupDomain.vue

```diff
@@ -126,7 +126,7 @@ onUnmounted(() => clearInterval(waitingForRegistrationInterval));
       />
 
       <BaseMessageBlock
-        v-if="defaultNetwork === '5'"
+        v-if="defaultNetwork && env === 'demo'"
```

Comment:
```suggestion
        v-if="env === 'demo'"
```
Do you need this check?